controllers/chat/sendSignedMessage.js

```diff
@@ -86,6 +87,8 @@ const sendSignedMesage = async (req, res) => {
     if (createdChannel?.channel?.is_channel_blocked)
       return res.status(403).json(responseError('Channel is blocked'));
 
+    let channelMember = [];
+    let senderInfo = [];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These variables are declared but not used until much later in the code. It would be better to declare them closer to where they're first used to improve readability and maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->